homeassistant/components/zwave_js/discovery.py

```diff
@@ -390,6 +390,53 @@ def get_config_parameter_discovery_schema(
         product_type={0x0003},
         primary_value=SWITCH_MULTILEVEL_CURRENT_VALUE_SCHEMA,
     ),
+    # Merten 507801 Connect Roller Shutter
+    ZWaveDiscoverySchema(
+        platform=Platform.COVER,
+        hint="window_shutter",
+        manufacturer_id={0x007A},
+        product_id={0x0001},
+        product_type={0x8003},
+        primary_value=ZWaveValueDiscoverySchema(
+            command_class={CommandClass.SWITCH_MULTILEVEL},
+            property={CURRENT_VALUE_PROPERTY},
+            endpoint={0, 1},
+            type={ValueType.NUMBER},
+        ),
+        assumed_state=True,
+    ),
+    # Merten 507801 Connect Roller Shutter.
+    # Disable endpoint 2, as it has no practical function
+    ZWaveDiscoverySchema(
+        platform=Platform.COVER,
+        hint="window_shutter",
+        manufacturer_id={0x007A},
+        product_id={0x0001},
+        product_type={0x8003},
+        primary_value=ZWaveValueDiscoverySchema(
+            command_class={CommandClass.SWITCH_MULTILEVEL},
+            property={CURRENT_VALUE_PROPERTY},
+            endpoint={2},
+            type={ValueType.NUMBER},
+        ),
+        assumed_state=True,
+        entity_registry_enabled_default=False,
+    ),
+    # Merten 507801 Connect Roller Shutter.
+    # Disable endpoint 2, as it has no practical function
+    ZWaveDiscoverySchema(
```

Comment:
Looks like a duplicate entry?

Reply:
No, as there is one for MultiLevelSwitch, and one for Protection.
I can specify in the comment, if you think it's needed.

Reply:
After looking at it, I guess I can merge these two into a single entry.
I'll fix it later today.

Reply:
You misunderstand me. You added three to the list when you only need two. The last two are dupes

Reply:
No problem!
The last two are for disabling endpoint 2 by default.
Both Protection and Switch Multilevel has to be disabled, as everything is doubled up with two endpoints on this silly device.